sql/updates/world/2015_12_09_12_world.sql

```diff
@@ -0,0 +1,40 @@
+--
+-- SAI script for Wyrmrest Temple taxi Flight Paths
+
+-- Ground ----> Top: 878 (26443,9455,0)
+-- Ground -> Middle: 883 (26443,9455,1)
+-- Top ----> Ground: 879 (26949,9457,0)
+-- Top ----> Middle: 880 (26949,9457,1)
+-- Middle ----> Top: 881 (27575,9563,0)
+-- Middle -> Ground: 882 (27575,9563,1)
+
+SET @Tariolstrasz := 26443;
+SET @Torastrasza  := 26949;
+SET @Afrasastrasz := 27575;
+
+-- Set AIName and remove old ScriptName in creature_template for all 3 NPCs
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `ScriptName`= '' WHERE `entry` IN (@Tariolstrasz,@Afrasastrasz,@Torastrasza);
+
+-- Tariolstrasz (Steward of Wyrmrest Temple) 26443,9455,878,883
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` = @Tariolstrasz;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(@Tariolstrasz, 0, 0, 1, 62, 0, 100, 0, 9455, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Tariolstrasz - On Gossip Option 0 Selected - Close Gossip'),
+(@Tariolstrasz, 0, 1, 0, 61, 0, 100, 0, 0, 0, 0, 0, 52, 878, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Tariolstrasz - On Gossip Option 0 Selected - Activate Taxi Path 878 Ground - Top'),
+(@Tariolstrasz, 0, 2, 3, 62, 0, 100, 0, 9455, 1, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Tariolstrasz - On Gossip Option 1 Selected - Close Gossip'),
+(@Tariolstrasz, 0, 3, 0, 61, 0, 100, 0, 0, 0, 0, 0, 52, 883, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 'Tariolstrasz - On Gossip Option 1 Selected - Activate Taxi Path 883 Ground - Middle');
```

Comment:
one possible solution to have only 3 sai entries per npc: on gossip select -> activate taxi -> link to close gossip